### DISCUSSION TITLE
Convolution EQs?

### DISCUSSION THREAD

**Discussion by jtrv** [pierreaubert/spinorama#243] [Q&A]
I am curious to know if the spinorama data could be used to create convolution EQ similar to [autoeq](https://autoeq.app/) or if speakers could benefit from convolution EQ vs parametric EQ? 

(Please excuse me if this is not a good place to ask or if it is a 'basic' question, I'm just a curious hobbyist)

**Comment by pierreaubert** [comment]
Hello,

I can generate a convolution EQ if you want. It will not be better than a
parametric EQ since we do not have the data for phase (only frequency). If
we also had
the phase we could optimise both at the same time with a FIR (convolution)
filter which is impossible to do with an IIR (parametric) filter.

On Sat, Jun 17, 2023 at 2:03 AM jtrv ***@***.***> wrote:

> I am curious to know if the spinorama data could be used to create
> convolution EQ similar to autoeq <https://autoeq.app/> or if speakers
> could benefit from convolution EQ vs parametric EQ?
>
> (Please excuse me if this is not a good place to ask or if it is a 'basic'
> question, I'm just a curious hobbyist)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/243>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH6U5WWEITNYAMUKN73XLTX4FANCNFSM6AAAAAAZJ3EGJE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
PfA

**Reply by jtrv** [reply]
Would the sort of impedance/phase measurements Amir and Erin use in their reviews apply here?

**Comment by pierreaubert** [comment]
Hi,

Sadly no. When you measure the signal (response of the loudspeaker), the
signal is of a complex value (as is the mathematical complex number). When
Amir published his data from the Klippel he only provides the frequency
information and not the phase one. They are related. If z is the measured
signal then z = spl * cos(phase) + j * spl * sin(phase) where j*j = -1.

The phase corrections that you can do have a subtle effect and most people
are not sensitive to them. That's why Amir (and others) published only the
SPL information which is the most important.

On Mon, Jun 19, 2023 at 5:52 AM jtrv ***@***.***> wrote:

> Would the sort of impedance/phase measurements Amir and Erin use in their
> reviews apply here?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/243#discussioncomment-6214409>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH4M4MVCHOFRYN3P7TTXL7EHDANCNFSM6AAAAAAZJ3EGJE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
--
PfA
